### ISSUE TITLE
How to use arrow keys?

### ISSUE THREAD

**Issue by jeyca** [klausw/hackerskeyboard#639]
I still didn't get it, how to use arrow keys. can I activate them somehow or do I have to press special buttons?

**Comment by TPS** [comment]
If the HK is open, they just work.  If you need to open HK when there's no text input, try the `Use Permanent Notification `:ballot_box_with_check: setting, which allows to open HK anytime.

**Comment by andrew-phi** [comment]
AFAIR, arrow/cursor keys are only available in full 5-row mode (choose in Input Settings).

**Comment by jeyca** [comment]
yes thats actually the problem what is bothering me, I really would like to use arrow keys in the small mode as well...

**Comment by TPS** [comment]
This is _precisely_ why there's a 5-row compact mode.…  Try it!

**Comment by yarda** [comment]
For me the 5 row compact layout is too small. So if I want to use the arrow keys in normal mode I have to rotate the device to landscape mode, which is not very comfortable and means slow down. Would be possible to add third, e.g. numeric keyboard layout with arrows, e.g. it could work the following way.:
- tap 123 key for numeric layout (this is how it works now)
- hold 123 key for just numeric keyboard layout with arrows and maybe some special keys

There could be different mechanism for switching instead of the 123 holding.

**Comment by TPS** [comment]
Maybe, perhaps, an _option_ to convert current kb's Fn layout to have large arrows in place of F1-F12?

**Comment by yarda** [comment]
It seems there are no FN keys in the normal portrait layout (or I don't know how to activate them). It would be great to have the FN keys together with the arrows somehow accessible from the normal portrait layout.

**Comment by yarda** [comment]
The no arrow keys in 4 row portrait layout is show stopper for me. With instant messaging I need often to move the cursor left/right. It's pretty simple with the Gboard, just swipe left/right on the spacebar. It seems hackerskeyboard has the option to configure actions to left/right swipe gestures, unfortunately it's impossible to map the left/right keys there.

For me it would be enough and it would improve the usability a lot, to allow mapping the arrow keys to the swipe gestures. Then I would map the left/right keys to the left/right swipe.
